tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# The Dockerfile's resulting image is used in executing Kepler Operator e2e tests on ProwCI
```

Comment:
So there is no escaping this :) ?

Reply:
@vprashar2929 could you please also 
* add a make target - `e2e-image`  that follows the same logic as`operator-build` 
* test the image can be built by adding this to the build-images jobs ?

Reply:
Yes definitely need that